### ISSUE TITLE
TIECon NW Event: Google Calendar == 400 Bad Request

### ISSUE THREAD

**Issue by eayoungs** [calagator/calagator#515]
http://calagator.org/events/1250471184

https://www.google.com/calendar/event?action=TEMPLATE&trp=true&text=TiECon+Northwest+%26+Pitch+Northwest&dates=20170126T160000Z%2F20170127T030000Z&location=The+Sentinel+Hotel%2C+614+Southwest+11th+Avenue%2C+Portland+OR+97205+US&sprop=website%3Awww.tieconnorthwest.org&details=Imported+from%3A+http%3A%2F%2Fcalagator.org%2Fevents%2F1250471184+%0A%0ATiE+Oregon%2C+TiE+Seattle%2C+TiE+Vancouver+BC+and+TiE+Rockies+are+coming+together+to+host+the+first-ever+TiECon+Northwest+in+Portland+Oregon+on+January+25-26th.+This+1.5+day+conference+will+bring+together+entrepreneurs+and+investors+and+will+highlight+three+disruptive+technology+sectors+%E2%80%93+Artificial+Intelligence%2FMachine+Learning%2C+Healthcare%2C+and+Augmented+Reality%2FVirtual+Reality%2FMixed+Reality.%0D%0A%0D%0AThe+possibilities+seem+huge+and+there+is+tremendous+hype%2C+full+of+great+buzzwords+and+slogans+but+for+investors+and+entrepreneurs%2C+key+questions+still+exist+regarding+these+technologies%3A%0D%0A%0D%0AWhat+is+the+value%3...[truncated]

**Comment by jhelwig** [comment]
Something to do with how the description is being truncated. Removing the `%3...[truncated]` from the end let the URL work for me.

**Comment by botandrose** [comment]
`...%5Btruncated%5D` perhaps?

**Comment by jhelwig** [comment]
Fortunately, browsers handle the `[` -> `%5D` automatically these days. Looks like it's the `%3` which is definitely invalid (needs 2 digits to be valid).

**Comment by jhelwig** [comment]
Yup. We're blindly truncating the fully-formed URL (https://github.com/calagator/calagator/blob/master/app/helpers/calagator/google_event_export_helper.rb#L11-L13), instead of the individual components that go into it (https://github.com/calagator/calagator/blob/master/app/helpers/calagator/google_event_export_helper.rb#L22-L24).
